packages/css/index.js

```diff
@@ -10,6 +12,6 @@ try {
 if (useTranspiled) {
   module.exports = require("./lib");
 } else {
-  require("babel/register");
```

Comment:
I haven't cleaned it up everywhere in this repo yet, but I've been moving away from using babel-register altogether.  The problem is that it can't be invoked twice (once for the plugin, once for another plugin, once for Interlock itself, etc).  So I've just been building in between tests - and, in the case of `interlock`, I have a `watch` command.

As more of a side note, we'll need to add `babel-register` to `package.json` if this line stays.